packages/components/src/base-control/styles/base-control-styles.js

```diff
@@ -12,11 +12,14 @@ import { space } from '../../ui/utils/space';
 export const Wrapper = styled.div`
 	font-family: ${ font( 'default.fontFamily' ) };
 	font-size: ${ font( 'default.fontSize' ) };
+	margin-bottom: ${ space( 2 ) };
```

Comment:
Now the Wrapper applies its margin bottom - I ensured that the last child of the wrapper doesn't have a bottom margin.
This allows consumers not to have to go inside the `Wrapper` to remove the spacing, when necessary.

I think ideally, the `Wrapper` would not have a `margin-bottom` at all, but this ensures backwards compatibility. Should I remove this?
```suggestion
```

Reply:
Given BaseControl isn't experimental we should ensure backwards compat, but this seems like a good improvement to exclude it in the right conditions.